ui/components/app/assets/nfts/nfts-tab/nfts-tab.js

```diff
@@ -84,6 +85,7 @@ export default function NftsTab() {
         referrer: ORIGIN_METAMASK,
       },
     });
+    endTrace({ name: TraceName.AccountOverviewNftsTab });
```

Comment:
@HowardBraham Had to remove this line as it always triggers when `nftsLoading` is set to false, even if the `detectNfts` background request is still pending. I'll use this to implement `TraceName.AccountOverviewNftsTabFCP` in the PR for additional spans.

```suggestion
```